src/System.Private.ServiceModel/src/windows/project.json

```diff
@@ -0,0 +1,64 @@
+{
+  "dependencies": {
+    "System.Collections": "4.0.11-beta-*",
+    "System.Collections.Concurrent": "4.0.11-beta-*",
+    "System.Collections.NonGeneric": "4.0.1-beta-*",
+    "System.Collections.Specialized": "4.0.1-beta-*",
+    "System.ComponentModel": "4.0.1-beta-*",
+    "System.ComponentModel.EventBasedAsync": "4.0.11-beta-*",
+    "System.Diagnostics.Contracts": "4.0.1-beta-*",
+    "System.Diagnostics.Debug": "4.0.11-beta-*",
+    "System.Diagnostics.Tools": "4.0.1-beta-*",
+    "System.Diagnostics.Tracing": "4.0.21-beta-*",
+    "System.Globalization": "4.0.11-beta-*",
+    "System.IO": "4.0.11-beta-*",
+    "System.IO.Compression": "4.1.0-beta-*",
+    "System.Linq": "4.0.1-beta-*",
+    "System.Linq.Expressions": "4.0.11-beta-*",
+    "System.Linq.Queryable": "4.0.1-beta-*",
+    "System.Net.Http": "4.0.1-beta-*",
+    "System.Net.Primitives": "4.0.11-beta-*",
+    "System.Net.WebHeaderCollection": "4.0.1-beta-*",
+    "System.Net.WebSockets": "4.0.0-beta-*",
+    "System.Net.WebSockets.Client": "4.0.0-beta-*",
+    "System.ObjectModel": "4.0.11-beta-*",
+    "System.Reflection": "4.0.11",
+    "System.Reflection.DispatchProxy": "4.0.1-beta-*",
+    "System.Reflection.Extensions": "4.0.1-beta-*",
+    "System.Reflection.Primitives": "4.0.1-beta-*",
+    "System.Reflection.TypeExtensions": "4.0.1-beta-*",
+    "System.Resources.ResourceManager": "4.0.1-beta-*",
+    "System.Runtime": "4.0.21-beta-*",
+    "System.Runtime.Extensions": "4.0.11-beta-*",
+    "System.Runtime.Handles": "4.0.1-beta-*",
+    "System.Runtime.InteropServices": "4.0.21-beta-*",
+    "System.Runtime.Serialization.Xml": "4.0.10",
+    "System.Security.Claims": "4.0.1-beta-*",
+    "System.Security.Cryptography.X509Certificates": "4.0.0-beta-*",
+    "System.Security.Principal": "4.0.1-beta-*",
+    "System.Text.Encoding": "4.0.11-beta-*",
+    "System.Threading": "4.0.11-beta-*",
+    "System.Threading.Tasks": "4.0.11-beta-*",
+    "System.Threading.Timer": "4.0.1-beta-*",
+    "System.Xml.ReaderWriter": "4.0.11-beta-*",
+    "System.Xml.XDocument": "4.0.11-beta-*",
+    "System.Xml.XmlDocument": "4.0.1-beta-*",
+    "System.Xml.XmlSerializer": "4.0.11-beta-*"
+  },
+  "frameworks": {
+    "dnxcore50": {
+      "dependencies": {
+        "System.Net.Http.WinHttpHandler": "4.0.0-beta-*",
+        "System.Net.NameResolution": "4.0.0-beta-*",
+        "System.Net.Security": "4.0.0-beta-*",
+        "System.Net.Sockets": "4.1.0-beta-*",
+        "System.Security.Principal.Windows": "4.0.0-beta-*"
+      }
+    },
+    "netcore50": {
+      "dependencies": {
+        "System.Runtime.WindowsRuntime": "4.0.11-beta-*"
+      }
+    }
+  }
+}
```

Comment:
Unfortunately no. There is no mechanism in project.json files to specify a dependency which only exists for a particular OS. This means we need to override the ProjectJson and ProjectLockJson build properties to pick up a different project.json file. This is a real pain, but will eventually be resolved in one of two ways. Either OS specific dependency support will be added (there is a bug open for this) or HttpClientHandler will have the appropriate API's needed so we don't need to use WinHttpHandler. This will be post rtm so unavoidable for now.